app/templates/components/goal/goal-overview.hbs

```diff
@@ -17,10 +17,10 @@ Maecenas nec pulvinar nulla, nec elementum tellus. Etiam porta tincidunt tellus,
     </div>
     <div class="goal_coret goal_right">
       <h2>The Big Four</h2>
-      <div id="PO" class="{{unless goal.po "red "}}circle">{{get-user goal.po}}</div>
-      <div id="HD" class="{{unless goal.hd "red "}}circle">{{get-user goal.hd}}</div>
-      <div id="HFE" class="{{unless goal.hfe "red "}}circle">{{get-user goal.hfe}}</div>
-      <div id="HBE" class="{{unless goal.hbe "red "}}circle">{{get-user goal.hbe}}</div>
+      <div class="{{unless goal.po "red "}}circle PO">{{get-user goal.po}} <p>Product Owner</p></div>
```

Comment:
I would separate this into it's own component that display's the core team of a goal. That way, it is reusable and cleaner.